portality/scripts/anon_import.py

```diff
@@ -113,10 +164,23 @@ def do_import(config):
     # once we've finished importing, clean up by deleting the entire temporary container
     tempStore.delete_container(container)
 
+    # create aliases for the indexes
+    print("\n==Creating Aliases==")
+    for index_detail in index_details.values():
+        for retry in range(5):
+            if not es_connection.indices.exists(index_detail.alias_name):
+                break
+            print(f"Old alias exists, waiting for it to be removed, alias[{index_detail.alias_name}] retry[{retry}]")
+            sleep(5)
+
+        print("Creating alias: {:<30} -> {}".format(index_detail.instance_name, index_detail.alias_name))
+        es_connection.indices.put_alias(index=index_detail.instance_name, name=index_detail.alias_name)
```

Comment:
It would be helpful to perform the alias creation earlier in the script - i.e. immediately after initialisation rather than after data import, because the site requires the aliases to function, and it takes a number of minutes to insert all of the data.

If we create aliases immediately after the records, the DOAJ app will run successfully straight away with empty data until the type is reached in import.